src/Framework/Migrations/Contracts/Migration.php

```diff
@@ -40,4 +40,15 @@ public static function id() {
 	public static function timestamp() {
 		throw new RuntimeException( 'This method must be overridden to return a valid unix timestamp' );
 	}
+
+	/**
+	 * Run the migration check
+	 *
+	 * @since 2.9.7
+	 *
+	 * @return bool
+	 */
+	public static function check() {
```

Comment:
This should be an instance method, not static.

Reply:
Resolved 
b2051ce